content/docs/self-hosted-runners.md

```diff
@@ -201,8 +201,7 @@ machine or on-premise GPU cluster.
 cml-runner \
   --repo="$repository_url" \
   --token="$personal_access_token" \
-  --labels="local,runner" \
-  --idle-timeout=0
+  --labels="local,runner"
```

Comment:
this change would be OK if we address https://github.com/iterative/cml/pull/700#issuecomment-902926901

Reply:
this probably should be reverted...

Reply:
👍🏼

Reply:
> _leave --idle-timeout=-1_

:this: